### PR TITLE
V4(Wallet options updates)

### PR DESCRIPTION
**Pull request by schnogz**
- Enables BCH send for Lockbox
- Adds unique `siteId`s for Matmomo analytics
- Updates Testnet announcement

### CODE REVIEW

File: testnet/wallet-options-v4.json [hunk 1]
```diff
@@ -2,28 +2,39 @@
...
+            ]
           },
           "wallet": {
-            "id": "testnet-warning-wallet",
+            "id": "testnet-warning",
             "type": "danger",
             "icon": "alert-filled",
             "header": {
-              "en": "Warning! This is the testnet3 blockchain. Testnet coins have no value."
+              "en": "TESTNET WARNING"
```
Comment (tuzzolo) [inline testnet/wallet-options-v4.json]: Why would we make this warning less informative?

Comment (schnogz) [reply testnet/wallet-options-v4.json]: The message is now in the body section. The header gets capitalized on the frontend and it was jarring.

Comment (schnogz) [reply testnet/wallet-options-v4.json]: see lines 30-37

File: testnet/wallet-options-v4.json [hunk 2]
```diff
@@ -2,28 +2,39 @@
...
             "icon": "alert-filled",
             "header": {
-              "en": "Warning! This is the testnet3 blockchain. Testnet coins have no value."
-            }
+              "en": "TESTNET WARNING"
+            },
+            "sections": [
+              {
+                "body": {
+                  "en": "This is the testnet3 blockchain. Your coins have no value here."
```
Comment (tuzzolo) [inline testnet/wallet-options-v4.json]: I'd prefer this language: "Warning! This is the testnet3 blockchain. Testnet coins have no value."

File: testnet/wallet-options-v4.json [hunk 3]
```diff
@@ -2,28 +2,39 @@
...
             },
-            "sections": [],
-            "action": {}
-          },
-          "sendBch": {},
-          "receiveBch": {}
+            "sections": [
+              {
+                "body": {
+                  "en": "This is the testnet3 blockchain. Your coins have no value here."
```
Comment (tuzzolo) [inline testnet/wallet-options-v4.json]: Same here: "Warning! This is the testnet3 blockchain. Testnet coins have no value."

File: testnet/wallet-options-v4.json (hunk 2 above)
Comment (schnogz) [reply testnet/wallet-options-v4.json]: 👍

**Comment by schnogz** [comment]
screen shot 2019-02-08 at 10 42 49 am

**Pull request merged**